src/css/flags/user-inactive.less

```diff
@@ -11,13 +11,20 @@
           cursor: none;
           -webkit-cursor-visibility: auto-hide;
       }
+      .jw-dock {
+        display: block;
```

Comment:
Why are you adding `display: block` here? Is it set to none somewhere else? If so that's where you remove none and replace it wit these visibility/opacity 0.